packages/commonwealth/server/routes/getNewProfile.ts

```diff
@@ -32,13 +32,21 @@ const getNewProfile = async (
   next: NextFunction
 ) => {
   const { profileId } = req.query;
-  if (!profileId) return next(new Error(Errors.NoIdentifierProvided));
+  let profile;
 
-  const profile = await models.Profile.findOne({
-    where: {
-      id: profileId,
-    },
-  });
+  if (profileId) {
```

Comment:
Is this ideal behavior? Ideally we should require the GET route to pass a profile id, even if a user is logged in.